code/game/jobs/job/central.dm

```diff
@@ -104,3 +104,62 @@
 	if(visualsOnly)
 		return
 	H.mind.offstation_role = TRUE
+
+
+// Mentor+ role for instructing new players
+/datum/job/ntinstructor
+	title = "Nanotrasen Instructor"
+	flag = JOB_CENTCOM
+	department_flag = JOB_CENTCOM
+	total_positions = 2
+	spawn_positions = 0
+	supervisors = "the admins"
+	selection_color = "#ffdddd"
+	access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_COURT, ACCESS_FORENSICS_LOCKERS,
+			            ACCESS_MEDICAL, ACCESS_ENGINE, ACCESS_EVA, ACCESS_HEADS,
+			            ACCESS_MAINT_TUNNELS, ACCESS_BAR, ACCESS_JANITOR, ACCESS_CONSTRUCTION, ACCESS_MORGUE,
+			            ACCESS_KITCHEN, ACCESS_CARGO, ACCESS_CARGO_BOT, ACCESS_MAILSORTING, ACCESS_QM, ACCESS_HYDROPONICS, ACCESS_LAWYER,
+			            ACCESS_THEATRE, ACCESS_CHAPEL_OFFICE, ACCESS_LIBRARY, ACCESS_RESEARCH, ACCESS_MINING, ACCESS_MINING_STATION,
+			            ACCESS_CLOWN, ACCESS_MIME, ACCESS_RC_ANNOUNCE, ACCESS_WEAPONS)
+	minimal_access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_COURT, ACCESS_FORENSICS_LOCKERS,
+			            ACCESS_MEDICAL, ACCESS_ENGINE, ACCESS_EVA, ACCESS_HEADS,
+			            ACCESS_MAINT_TUNNELS, ACCESS_BAR, ACCESS_JANITOR, ACCESS_CONSTRUCTION, ACCESS_MORGUE,
+			            ACCESS_KITCHEN, ACCESS_CARGO, ACCESS_CARGO_BOT, ACCESS_MAILSORTING, ACCESS_QM, ACCESS_HYDROPONICS, ACCESS_LAWYER,
+			            ACCESS_THEATRE, ACCESS_CHAPEL_OFFICE, ACCESS_LIBRARY, ACCESS_RESEARCH, ACCESS_MINING, ACCESS_MINING_STATION,
+			            ACCESS_CLOWN, ACCESS_MIME, ACCESS_RC_ANNOUNCE, ACCESS_WEAPONS)
+	mentor_only = TRUE
+	transfer_allowed = FALSE
+	outfit = /datum/outfit/job/ntinstructor
+
+/datum/outfit/job/ntinstructor
+	name = "Nanotrasen Instructor"
+	jobtype = /datum/job/ntinstructor
+
+	uniform = /obj/item/clothing/under/rank/centcom/representative // formal NT uniform
+	shoes = /obj/item/clothing/shoes/centcom // fancy shoes
+	head = /obj/item/clothing/head/beret/blue // generic blue beret
+	l_ear = /obj/item/radio/headset/heads/ntinstructor //
+	glasses =  /obj/item/clothing/glasses/sunglasses
+	id = /obj/item/card/id/nanotrasen
+	belt = /obj/item/storage/belt/utility/full/multitool
+	pda = /obj/item/pda/heads/ntrep
+	implants = list(
+		/obj/item/implant/mindshield
+	)
+	backpack = /obj/item/storage/backpack/satchel
+	backpack_contents = list(
+		/obj/item/melee/classic_baton/telescopic = 1,
+		/obj/item/pinpointer/crew = 1,
+		/obj/item/healthanalyzer/advanced = 1,
+		/obj/item/laser_pointer/blue = 1
+
+	)
+	box = /obj/item/storage/box/centcominstructor
+
+/datum/outfit/job/ntinstructor/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```

Comment:
I actually considered doing this. But I decided it would cause more problems than it would solve. For example, what if they die and are cloned? They can't re-equip their old items. What if they're gibbed? Someone picks up their ID and cannot drop it. What if they decide there are not enough people who need instruction and they want to transfer to another job? They can't hand over their ID for job transfer. Etc.

Reply:
The getting gibbed won't be an issue due to the antidrop being removed if it's not set on the item initialy. Aka if the item doesn't have it defined then it won't have it on dropped.

The other two points I can understand yeah. It depends on how you want the job to take form. Is it an assistant that is actually trust worthy? Or somebody from CC that is there to just teach and not interact much with the station otherwise.

In terms of cryoing and such. You can maybe make it so that if they are cryod the players are taken back to the lobby.